server/src/prisma/config/validations.js

```diff
@@ -0,0 +1,14 @@
+import allValidations from "../validations/index.js"
+
+const validations = (nameOfModel) =>  {
+    const schema = {}
+    for (let schemaName in allValidations) {
+        const modelName = allValidations[schemaName].modelName
```

Comment:
@nwalberts To the best of my knowledge...

```
import { globallyScopedThing } from 'thing';
function stuff()
   const localizedThing  = globallyScopedThing
   // use localizedThing
}
```

This works because localized is only a reference that is then garbage collected at the end of function execution